components/GameScreen/GeneratedMusic.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react"
+import { Contract, ethers } from "ethers"
+import Image from "next/image"
+import abi from "../../lib/abi-musicGame.json"
+import getIpfsLink from "../../lib/getIpfsLink"
+
+const GeneratedMusic = ({ checked, choices, onClickHandler }: any) => {
+  const [trackMetadata, setTrackMetadata] = useState([])
+
+  const getProvider = () => {
+    const goerliRpc = "https://ethereum-goerli-rpc.allthatnode.com"
```

Comment:
should move to env maybe?

Reply:
updated to
- use ENV variable `NEXT_PUBLIC_RPC_URL`
- abstracted to a util: `getProvider.ts`

![Screenshot 2023-02-28 at 4 30 56 PM](https://user-images.githubusercontent.com/23249402/221959068-c861223c-8269-4668-b1fe-ec54835c4ef8.png)